cdc/capture/http_handler.go

```diff
@@ -798,7 +742,17 @@ func (h *HTTPHandler) forwardToOwner(c *gin.Context) {
 	defer resp.Body.Close()
 	_, err = bufio.NewReader(resp.Body).WriteTo(c.Writer)
 	if err != nil {
-		c.IndentedJSON(http.StatusInternalServerError, model.NewHTTPError(err))
+		handleError(c, err)
 		return
 	}
 }
+
+func handleError(c *gin.Context, err error) {
```

Comment:
could this be built as a ErrorHandle middleware. FYI https://stackoverflow.com/questions/48224908/better-error-handling